prow/config.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-# Copyright 2021 The Knative Authors
+# Copyright 2022 The Knative Authors
```

Comment:
I think the rule is if we are only modifying the file, no need to change the copyright year.

Reply:
Ok so only when we are changing from `Google` to `Knative Authors`?

Reply:
Yep